tests/features/step_definitions/incidents_steps.coffee

```diff
@@ -23,7 +23,6 @@ do ->
       # Location
       @client.setValue('input.select2-search__field', 'f')
       @client.clickWhenVisible('.select2-results__option--highlighted')
-      @client.pause(40000)
```

Comment:
I would try leaving in a pause here, although I don't think it should be 40 seconds.